ginga/cmap.py

```diff
@@ -13309,8 +13309,7 @@ def add_matplotlib_cmap(cm, name=None):
 def add_matplotlib_cmaps(fail_on_import_error=True):
     """Add all matplotlib colormaps."""
     try:
-        import matplotlib.pyplot as plt
-        from matplotlib import cm as _cm
+        from matplotlib import colormaps as _mpl_cm
         if MPL_LT_3_4:
             from matplotlib.cbook import mplDeprecation as MatplotlibDeprecationWarning
         else:
```

Comment:
@pllim, should we drop support for mpl < 3.4 and git rid of this workaround needed for suppressing `MatplotlibDeprecationWarning` ?

Reply:
Matplotlib is at 3.8 now.  Matplotlib 3.3 has not been touched since early 2021. Do you still have any users stuck with matplotlib < 3.4? If not, probably less work to drop it.

Reply:
Sounds good. I'll remove the support from this module for that.

Reply:
Should also bump this? 

https://github.com/ejeschke/ginga/blob/f0eecdf92eb1b3955482240aa6fd957aa7f807e4/setup.cfg#L60

Reply:
good catch